pkg/storage/internalstorage/util.go

```diff
@@ -19,9 +20,27 @@ import (
 const (
 	SearchLabelFuzzyName = "internalstorage.clusterpedia.io/fuzzy-name"
 
-	URLQueryWhereSQL = "whereSQL"
+	URLQueryWhereSQL           = "whereSQL"
+	URLQueryWhereSQLParameters = "whereSQLParameters"
```

Comment:
In order to make it clearer to the user that the sql parameterizes the query, we might use a new query name instead of whereSQL.

`whereSQL` can also be considered deprecated